iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Branch.java

```diff
@@ -234,19 +236,49 @@ default ApparentPowerLimits getNullableApparentPowerLimits1() {
      * are valid.</p>
      * @param currentLimits a set of existing current limits.
      * @return an adder allowing to create a new {@link CurrentLimits} initialized from the limits in parameters in the selected {@link OperationalLimitsGroup} on side 1.
+     * @deprecated Use {@link OperationalLimitsGroup#newCurrentLimits(CurrentLimits)} instead.
      */
+    @Deprecated(since = "6.8.0")
     default CurrentLimitsAdder newCurrentLimits1(CurrentLimits currentLimits) {
         CurrentLimitsAdder currentLimitsAdder = newCurrentLimits1();
         return initializeFromLoadingLimits(currentLimitsAdder, currentLimits);
     }
 
+    /**
+     * <p>Get the {@link OperationalLimitsGroup} corresponding to the default ID or create a new one if it does not exists.
```

Comment:
```suggestion
     * <p>Get the {@link OperationalLimitsGroup} corresponding to the default ID or create a new one if it does not exist.
```
There are some other places to check to remove this "s".